process/transaction/shardProcess.go

```diff
@@ -353,32 +353,27 @@ func (txProc *txProcessor) processTxFee(
 	}
 
 	moveBalanceFee := txProc.economicsFee.ComputeMoveBalanceFee(tx)
+	totalCost := txProc.economicsFee.ComputeTxFee(tx)
```

Comment:
Are we taking into account not to produce an invalid receipt for the following situation?
Account balance: 1 eGLD, move balance tx with gasLimit * gasPrice = 150eGLD? We should have a test for this scenario checking the receipt's value not to be around 135eGLD.

Reply:
the code will not reach here if there is not enough balance in the account. it will stop at checktxvalues

Reply:
Added test, the code performs as designed :+1: